src/scribe.js

```diff
@@ -47,6 +47,7 @@ define([
     this.options = defaults(options || {}, {
       allowBlockElements: true,
       debug: false,
+      shouldUndo: true,
```

Comment:
Im not sure thats necessary as there are no more configuration options to pass.  If there were then I'd be up for this.